### PR TITLE
Renames all mentions of Solfed to TerraGov

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Renames SolFed/SolGov/Solar Goverment/Solar Federation to TerraGov/Terran Goverment

## Why It's Good For The Game
Requested by the loremins

## Proof Of Testing
It compiles, nothing seems broken ingame (please TM this)

</details>

## Changelog

:cl:
add: Renames all mentions of the Solar Goverment/Federation to the Terran Goverment
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
Does not rename the Sol calibers by request of @okithedoki

File: code/modules/events/market_crash.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	var/list/poss_reasons = list("the alignment of the moon and the sun",\
 		"some risky housing market outcomes",\
 		"the B.E.P.I.S. team's untimely downfall",\
-		"speculative SolFed grants backfiring",  /*SKYRAT EDIT CHANGE; original was "speculative Terragov grants backfiring"*/\
+		"speculative TerraGov grants backfiring",  /*SKYRAT EDIT CHANGE; original was "speculative Terragov grants backfiring"*/\
```
Comment (ReturnToZender) [inline code/modules/events/market_crash.dm:26]: You can remove the Skyrat Edit marker if this is just restoring it to its original state.

**Comment by ReturnToZender** [comment]
Query as to whether "Terran State(s)" may be better for the lore to make it distinct as we flesh out Orion and Adasta, but that's sorta out of scope for this

**Comment by Arturlang** [comment]
todo update path changes with Updatepaths

**Comment by Majkl-J** [comment]
> todo update path changes with Updatepaths (if neccesary)

Once those are done it's ready to merge

**Comment by KazooBard** [comment]
FUCK YEAH TERRAGOV WOOHOO BABY LETS GO THIS IS THE LORE I'VE BEEN WAITING FOR, TGMC ARE CANON

**Comment by sippykot** [comment]
don't like it :/ 
if anything it should be the reverse
"sol" was way more established than "terra"

**Comment by Arturlang** [comment]
Did filepath

**Comment by Arturlang** [comment]
@sippykot I suggest talking with the lore people on the discord, prefferably in the lroe channel

**Comment by Arturlang** [comment]
Ready to go

**Comment by projectkepler-ru** [comment]
Did we ever held a second poll about what name for the central government?

**Comment by Arturlang** [comment]
> Did we ever held a second poll about what name for the central government?

I have no idea honestly, I don't remember one being held

**Pull request merged**